lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -343,4 +353,39 @@ public function mergeClusters(array $oldCluster, array $newCluster): array {
 		}
 		return $result;
 	}
+
+	private function fillFaceRelationsFromPersons(string $userId) {
+		$deviation = $this->settingsService->getDeviation();
+		if (!version_compare(phpversion('pdlib'), '1.0.2', '>=') || ($deviation === 0.0))
```

Comment:
Thinking aloud... should you put some print statement here? So, we can see from logs that it is not getting executed?

Reply:
Maybe.. Probably in the next nightly version keep these guards.
But I am seriously thinking to indicate that 1.0.2 will be necessary for the first public release.